ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -439,3 +455,32 @@
     (if (not (nil? first-matching-item))
       true
       false)))
+
+(defn clean-up-csv-value [value]
+  (when-not (nil? value)
+    (str/trim
+      (str/replace value "\"" ""))))
+
+(defn valid-business-id? [value]
+  (let [pattern  #"\d{7}-\d"]
+    (boolean (re-matches pattern value))))
+
+(defn parse-csv-response->company-map
+  "Convert given vector to maps of business-id and company names."
+  [csv-data]
+  (let [headers (first csv-data)
+        companies (map (fn [[business-id name]]
+           {::t-service/business-id (clean-up-csv-value business-id)
+            ::t-service/name        (clean-up-csv-value name)})
+         (rest csv-data))
+        valid-companies (filter #(valid-business-id? (::t-service/business-id %)) companies)]
+    valid-companies))
```

Comment:
This validation and cleanup seems very similar to the backend code, maybe we should make a single shared .cljc namespace that handles both cases.